health-services/project-factory/src/server/utils/excelUtils.ts

```
@@ -235,8 +235,121 @@ function addDataToSheet(
+      // Hide the column if specified
+      if (schema?.properties[property]?.hideColumn) {
+        sheet.getColumn(currentColumnIndex).hidden = true;
+      }
+    }
+  }
+}
+
+// -----------------------------
+// Function to Apply Dropdowns
+// -----------------------------
+function applyDropdownsForMultiSelect(sheet: any, currentColumnIndex: number, maxSelections: number, enumsList: string[], isChildOfSomeCampaign: boolean = false, rowsLength: number = 1) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)